tests/console/yast2_kdump_setup.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Prepare for yast2 kdump test, include initial test and install packages needed.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+
+use registration qw(add_suseconnect_product remove_suseconnect_product);
+use testapi qw(select_console);
+use utils;
```

Comment:
let's be explicit as I mentioned in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16072#discussion_r1055138543
otherwise we forget always unnecessary imports:
`use utils qw(zypper_call)` and remove import of `is_sle`.
Please apply all the files if more occurrences.